[Excel] (ExcelApi 1.16) Promote conceptual docs from preview to GA (#3685)
OfficeDev/office-js-docs-pr: docs/excel/excel-data-types-overview.md

---
title: Overview of data types in Excel add-ins
description: Data types in the Excel JavaScript API enable Office Add-in developers to work with formatted number values, web images, entities, arrays within entities, and enhanced errors as data types.
ms.date: 10/10/2022
ms.topic: conceptual
ms.prod: excel
ms.custom: scenarios:getting-started
ms.localizationpriority: high
---

# Overview of data types in Excel add-ins

> Data types in the Excel JavaScript API enable add-in developers to organize complex data structures as objects, such as formatted number values, web images, and entities as [entity cards](excel-data-types-entity-card.md).

Reviewer: Quick suggestion for brevity to take or leave.
```suggestion
Data types organize complex data structures as objects. This includes formatted number values, web images, and entities as [entity cards](excel-data-types-entity-card.md).
```
